src/macros.rs

```diff
@@ -1003,32 +1028,32 @@ fn wrap_macro_args_inner(
     }
 
     if !use_multiple_lines && result.len() >= shape.width {
-        None
+        // This error won't be propagated since we will retry wrapping args in multiple lines
+        Err(RewriteError::Unknown)
     } else {
-        Some(result)
+        Ok(result)
     }
 }
 
 // This is a bit sketchy. The token rules probably need tweaking, but it works
 // for some common cases. I hope the basic logic is sufficient. Note that the
 // meaning of some tokens is a bit different here from usual Rust, e.g., `*`
 // and `(`/`)` have special meaning.
-//
-// We always try and format on one line.
-// FIXME: Use multi-line when every thing does not fit on one line.
 fn format_macro_args(
```

Comment:
I removed this comment (written in 2018) since rustfmt already use multi-line when macro args do not fit on one line. 
See `wrap_macro_args` 
```rust
wrap_macro_args_inner(context, args, shape, false)
        .or_else(|_| wrap_macro_args_inner(context, args, shape, true))
```
Please tell me if we still need to keep this comment.